packages/wallets/src/AuthProvider.tsx

```diff
@@ -4,7 +4,21 @@ import { AuthProvider } from '@deriv/api-v2';
 const WALLETS_LOGINID_LOCALSTORAGE_KEY = 'active_wallet_loginid';
 
 const WalletsAuthProvider = ({ children, ...rest }: Omit<ComponentProps<typeof AuthProvider>, 'loginIDKey'>) => (
-    <AuthProvider {...rest} loginIDKey={WALLETS_LOGINID_LOCALSTORAGE_KEY}>
+    <AuthProvider
+        {...rest}
+        loginIDKey={WALLETS_LOGINID_LOCALSTORAGE_KEY}
+        selectDefaultAccount={accountsObject => {
+            const loginIds = Object.keys(accountsObject);
+            const result = loginIds.filter((loginId: string) => {
+                const { account_category: accountCategory, account_type: accountType } = accountsObject[loginId];
+                const isWallet = accountCategory == 'wallet';
+                const isFiat = accountType == 'doughflow';
+                return isWallet && isFiat;
+            })[0];
+
+            return result;
+        }}
+    >
```

Comment:
@nijil-deriv 
do we really need this implementation here?
could we just move this function to AuthProvider.tsx?

Reply:
Do you mean the AuthProvider.tsx inside api-v2?

Reply:
yes

Reply:
though it could be moved directly to App.tsx and this file (packages/wallets/src/AuthProvider.tsx) could be removed as it does not seem to be really doing much / taking any responsibilities